docs/guides/audio-config.md

```diff
@@ -20,33 +22,18 @@ If not present, you'll have to update your bootup kernel params:
 
 # Audio Configuration Files
 
-In most scenarios, you should use [these files](https://gist.github.com/MCMrARM/c357291e4e5c18894bea10665dcebffb), following the instructions in that gist's `README.md`.
-
-Special scenarios are:
-
-- [2019 16" MacBook Pro audio files](https://gist.github.com/kevineinarsson/8e5e92664f97508277fefef1b8015fba) - that laptop has 6 speakers and needs slightly different config
-- [2020 13" MacBook Air audio files](https://gist.github.com/bigbadmonster17/8b670ae29e0b7be2b73887f3f37a057b)
-
-# Using Pipewire instead of Pulseaudio
-
-You'll need to modify the `/lib/udev/rules.d/91-pulseaudio-custom.rules` file from the links above.
-
-```diff
-SUBSYSTEM!="sound", GOTO="pulseaudio_end"
-ACTION!="change", GOTO="pulseaudio_end"
-KERNEL!="card*", GOTO="pulseaudio_end"
-
--SUBSYSTEMS=="pci", ATTRS{vendor}=="0x106b", ATTRS{device}=="0x1803", ENV{PULSE_PROFILE_SET}="apple-t2.conf"
-+SUBSYSTEMS=="pci", ATTRS{vendor}=="0x106b", ATTRS{device}=="0x1803", ENV{PULSE_PROFILE_SET}="apple-t2.conf", ENV{ACP_PROFILE_SET}="apple-t2.conf"
+Simply run the following to set up audio :-
 
-LABEL="pulseaudio_end"
+```bash
+git clone https://github.com/kekrby/t2-better-audio.git
+cd t2-better-audio
+./install.sh
```

Comment:
does this need sudo?

Reply:
it doesn't https://github.com/kekrby/t2-better-audio/blob/2d835c6e3d4fb0406d2933638d380c8b7fb92700/install.sh#L8

Reply:
Wdym?

Reply:
I mistakenly thought/remembered the script would say "please run me with sudo" and exit 1, but it doesn't do that so it's not an issue